src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -186,6 +186,8 @@ protected override bool OnShutdown(int timeoutMilliseconds)
             this.shutdownDrainTarget = this.circularBuffer.AddedCount;
             this.shutdownTrigger.Set();
 
+            OpenTelemetrySdkEventSource.Log.DroppedExportProcessorItems(this.droppedCount);
```

Comment:
Couple suggestion which I think will make this more useful:
1. As this is shared for LogRecord, Activity, we need to let use know which telemetry is dropped. Consider including tyeof(T) or similar.
2. If droppedCount is non-zero, we should be logging at higher severity than informational. We can either detect this here and call different methods, or in the actual EventSource class itself (by using a non-event, which inturn calls Info or Warning one based on droppedCount)

Reply:
Perhaps also adding some indication which processor instance is dropping telemetry? (e.g. in case there are multiple batch exporting processors).

Reply:
it may be hard.. I think we should be able to atleast include the nameof(exporter) this batchexportprocesor was exporting to.

Reply:
Yeah, something like the name of the exporter and the instance id (or object hash).

E.g. "ZipkinExporter@0x12345678 dropped 100 items on the floor!".